tests/test_pandas_api.py

```diff
@@ -1810,6 +1810,15 @@ def test_pandas_max(q):
     for i in range(100):
         assert float(qmax[i]) == float(pmax[i])
 
+    ktab = q('{1!x}', tab)

```

Comment:
```suggestion
    ktab = tab.set_index('sym')
```

We are trying to use the pandas APIs as much as we can in all documentation and tests.
Better for users and catches more bugs for us.

Reply:
It makes sense to use it with tests, but I was unsure about invoking it from the very pandas api itself, as you do with `values` above. But ok, I note it for future contributions!

Reply:
Yes for tests it's as much as possible but in the API itself it's more-case-by-case depending on the exact usage a bit more.